src/testdir/test_ins_complete.vim

```diff
@@ -2599,7 +2630,7 @@ func Test_complete_fuzzy_match()
   unlet g:word
 endfunc
 
-" Check that tie breaking is stable for completeopt+=fuzzy (which should
+" Check that tie breaking is stable for completeopt+=fuzzycollect (which should
```

Comment:
?

Reply:
emm this test can be safe remove right ? sry for typo

Reply:
Why remove?

Reply:
fuzzy not work for this keyword compleiton mode.   It seems a bit redundant. but there is no harm in keeping it

Reply:
um.. you may mis-understand, `fuzzy` of completeopt is to support for (filter) all kinds compl included keyword, vs/but to e.g keyword compl mode maybe not recommend using `fuzzycollect`.

Reply:
um.. you may mis-understand, `fuzzy` of completeopt is to support for (filter) all kinds compl included keyword, vs/but to e.g keyword compl mode maybe not recommend using `fuzzycollect`.